contracts/leverage_trading/src/create_order.rs

```diff
@@ -251,6 +250,103 @@ impl Contract {
         let order: Order = serde_json::from_str(order.as_str()).unwrap();
         self.add_or_update_order(&account_id, order, order_id);
     }
+
+    #[payable]
+    pub fn create_take_profit_order(
```

Comment:
Should verify:
1. if User owns the position and this position is opened (partially implemented)
2. if no take-profit orders exist